test/Sentry.AspNetCore.Tests/Extensions/HttpContextExtensionsTests.cs

```diff
@@ -8,14 +8,50 @@ namespace Sentry.AspNetCore.Tests.Extensions
 {
     public class HttpContextExtensionsTests
     {
-        private static void AddRouteValuesIfNotNull(RouteValueDictionary route, string key, string value)
+        private class Fixture
         {
-            if (value is not null)
+            public HttpContext GetSut(string pathBase = null)
             {
-                route.Add(key, value);
+                var httpContext = new DefaultHttpContext();
+                if (pathBase is not null)
+                {
+                    // pathBase must start with '/' otherwise the new PathString will throw an exception.
+                    httpContext.Request.PathBase = new PathString(pathBase);
+                }
+                return httpContext;
+            }
+
+            public HttpContext GetMvcSut(string area = null, string controller = null, string action = null
+                , string pathBase = null)
```

Comment:
If they are too long for one line, please stack them up:

```suggestion
            public HttpContext GetMvcSut(
                string area = null,
                string controller = null,
                string action = null,
                string pathBase = null)
```